kirkstone-poky-script.bash

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -x -e
+
+export PLATFORM_BRANCH="hmx-mickledore"
+export MANIFEST_BASE=kirkstone
+export DISTRO=poky
+export MACHINES="\
+  imx8mp-var-dart-hmx1 \
+  mx4-c61 \
+"
+export IMAGES=meta-hostmobility-image
+export TARGET_IMAGE="console-hostmobility-image"
+export DL_DIR=/media/storage_hdd/YOCTO_DOWNLOADS
+export DIR_WORK=${PWD}/${MANIFEST_BASE}
+
+git checkout ${PLATFORM_BRANCH}
+
+# BITBAKE! TODO If manifest name is changed from kirkstone.xml, change it below too TODO
+# TEMPORARY CHANGE remote -s , TODO: CHANGE BACK
+for MACHINE in $MACHINES; do
+  bygg_args="\
+    --delete-conf \
+    --build-tag $BUILD_TAG \
+    --manifest-file ${MANIFEST_BASE}.xml \
+    --distro ${DISTRO} \
+    --machine $MACHINE"
+  
+  scripts/bygg $bygg_args --repo-sync -- echo 'Repo sync done'
+  ## START AFTER SYNC SPECIAL BUILD INSTRUCTIONS TESTING ONLY
+  # (HEAD -> remove-meta-varscite-sdk)
+  # commit b33c5864626b1eae7ff89f135281143024e88e45
+  git -C $MANIFEST_BASE/sources/meta-mobility-poky-distro checkout b33c5864626b1eae7ff89f135281143024e88e45
+  
+  ## END   AFTER SYNC SPECIAL BUILD INSTRUCTIONS TESTING ONLY
+  
+  scripts/bygg $bygg_args -- bitbake console-hostmobility-image
+  
+  # If machine does not start with mx4, we skip the deploy script 
+  if [[ ! $MACHINE =~ ^mx4- ]]; then
+    continue
+  fi
+
+# SCRIPT RUN INSIDE DEPLOY DOCKER BELOW
+
```

Comment:
Can the deploy script handle multiple builds?

Reply:
yes